src/lib/ical/common.ts

```diff
@@ -130,3 +130,10 @@ export const sortByDate = () => {
 export const filterIfPassed = (now: number, offset: number = 0) => {
 	return ({ date }: { date: Date }) => date.valueOf() + offset > now;
 };
+
+
+export const cleanSummary = (summary?: string) => {
+  if(summary === undefined)
+		return "Unnamed Event";

```

Comment:
To conform with our style guidelines, I propose we format the code like so:

```suggestion
  if (summary === undefined) return "Unnamed Event";

```

Reply:
Ah I see. Sounds good to me.